vale/styles/jargonLint/README.md

```diff
@@ -1,9 +1,14 @@
-# jargonList rules
+## jargonLint rules
 
-## Tooling
+These rules are intended for general use and are written to complement other style guides.
 
-To sort rules that extend substitution
+### Descriptions
 
-```shell
-awk -FS=':' '{print $2 $1}' rule.yaml | sort | awk -FS=':' '{print $2 $1}'
-```
\ No newline at end of file
+**Licenses.yml**: Corrects miswritten license abbreviation and versions.
+
+### Sources, Tools, and Methodology
+
+You may reach the detailed explaination and sources for each rule in their own folder inside the [sources](/sources) directory.
+
+The folder for each rule:
+- Licenses.yml: [Licenses](/sources/Licenses)
```

Comment:
This is my suggestion on how to link to the sources directory and its subdirectories.

Reply:
@ccoVeille is this okay with you?